docs/content.zh/docs/dev/table/sql/queries/window-deduplication.md

```diff
@@ -0,0 +1,93 @@
+Therefore, window Deduplication queries have better performance if users don't need results updated per record. Usually, Window Deduplication is used with [Windowing TVF]({{< ref "docs/dev/table/sql/queries/window-tvf" >}}) directly. Besides, Window Deduplication could be used with other operations based on [Windowing TVF]({{< ref "docs/dev/table/sql/queries/window-tvf" >}}), such as [Window Aggregation]({{< ref "docs/dev/table/sql/queries/window-agg" >}}), [Window TopN]({{< ref "docs/dev/table/sql/queries/window-topn">}}) and [Window Join]({{< ref "docs/dev/table/sql/queries/window-join">}}). 
+
+Window Deduplication can be defined in the same syntax as regular Deduplication, see [Deduplication documentation]({{< ref "docs/dev/table/sql/queries/deduplication" >}}) for more information.
+Besides that, Window Deduplication requires the `PARTITION BY` clause contains `window_start` and `window_end` columns of the relation applied [Windowing TVF]({{< ref "docs/dev/table/sql/queries/window-tvf" >}}) or [Window Aggregation]({{< ref "docs/dev/table/sql/queries/window-agg" >}}).
+Otherwise, the optimizer won’t be able to translate the query.
+
+Flink uses `ROW_NUMBER()` to remove duplicates, just like the way of [Window Top-N query]({{< ref "docs/dev/table/sql/queries/window-topn" >}}). In theory, Window Deduplication is a special case of Window Top-N in which the N is one and order by the processing time or event time.
+
+The following shows the syntax of the Window Deduplication statement:
+
+```sql
+SELECT [column_list]
+FROM (
+   SELECT [column_list],
+     ROW_NUMBER() OVER (PARTITION BY window_start, window_end [, col_key1...]
+       ORDER BY time_attr [asc|desc]) AS rownum
+   FROM table_name) -- relation applied windowing TVF
+WHERE rownum = 1 [AND conditions]
+```
+
+**Parameter Specification:**
+
+- `ROW_NUMBER()`: Assigns an unique, sequential number to each row, starting with one.
+- `PARTITION BY window_start, window_end [, col_key1...]`: Specifies the partition columns which contain `window_start` and `window_end` columns i.e. the window deduplicate key.
+- `ORDER BY time_attr [asc|desc]`: Specifies the ordering column, it must be a [time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}). Currently Flink supports [processing time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}#processing-time) and [event time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}#event-time). Ordering by ASC means keeping the first row, ordering by DESC means keeping the last row.
+- `WHERE rownum = 1`: The `rownum = 1` is required for Flink to recognize this query is deduplication.
+
+{{< hint info >}}
+Note: the above pattern must be followed exactly, otherwise the optimizer won’t be able to translate the query.
```

Comment:
`the optimizer won’t be able to translate the query` -> `the optimizer won’t be able to translate the query to window deduplication`